### PR TITLE
Update cses-1196.mdx

### PR DESCRIPTION
**Pull request by aashishpiitk**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
> I have followed the code style guidelines mentioned

well, have you?

**Review by nwatx — CHANGES_REQUESTED** [review]
Read: https://usaco.guide/general/adding-solution/#code-conventions

**Comment by aashishpiitk** [comment]
> Read: https://usaco.guide/general/adding-solution/#code-conventions

Is it ok now?

File: solutions/cses-1196.mdx [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+---
+id: cses-1196
+source: CSES
+title: Flight Routes
+author: Aashish Patel
+---
+
+**Time Complexity:** $\mathcal{O}(mk\log (mk))$
+
+```cpp
```
Comment (TruaShamu) [inline solutions/cses-1196.mdx:11]:
```suggestion
<LanguageSection>
<CPPSection>

```cpp
```

Format with opening LanguageSection and CPPSection please

File: solutions/cses-1196.mdx [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
+{
+   ios_base:: sync_with_stdio(false); cin.tie(0);
+	
+	int t=1;
+	while(t--){
+		solve();
+	}
+   return 0;
+}
+```
```
Comment (TruaShamu) [inline solutions/cses-1196.mdx:72]: Add closing tags
```
</CPPSection>
</LanguageSection>
```
After line 72 please. (The triple backtick)

File: solutions/cses-1196.mdx [hunk 3]
```diff
@@ -1,3 +1,77 @@
+Solution 1:
+---
+id: cses-1196
+source: CSES
+title: Flight Routes
+author: Aashish Patel
```
Comment (TruaShamu) [inline solutions/cses-1196.mdx:6]:
```suggestion
author: Aashish Patel, Benjamin Qi
```

File: solutions/cses-1196.mdx [hunk 4]
```diff
@@ -1,3 +1,77 @@
+Solution 1:
```
Comment (TruaShamu) [inline solutions/cses-1196.mdx:1]: Move ```Solution 1:``` to after the section with the problem information (id, source, title, author)

File: solutions/cses-1196.mdx [hunk 5]
```diff
@@ -1,3 +1,77 @@
...
+	int t=1;
+	while(t--){
+		solve();
+	}
+   return 0;
+}
+```
+
+Solution 2:
 ---
```
Comment (TruaShamu) [inline solutions/cses-1196.mdx:75]: Remove this part, please
```
---
id: cses-1196
source: CSES
title: Flight Routes
author: Benjamin Qi
---
```

**Review by TruaShamu — CHANGES_REQUESTED** [review]
Some changes, I hope you won't mind addressing them.
It's likely that the deployment will succeed after this is done, if not, I'll review again.

For some references, feel free to look at this [link](https://usaco.guide/general/adding-solution/#code-conventions) and this [link](https://usaco.guide/general/adding-solution#steps) as well.

File: solutions/cses-1196.mdx [hunk 6]
```diff
@@ -2,14 +2,94 @@
...
+#define endl "\n"
+#define MOD 1000000007
+#define setprec(ans)  cout<<fixed<<std::setprecision(9)<<ans<<endl;
+typedef vector<int> VI;
+typedef vector<VI> VVI;
+typedef long long int ll;
+typedef vector<ll> VL;
+typedef vector<VL> VVL;
+typedef vector<pair<int, int>> VP;
+typedef vector<VP> VVP;
```
Comment (nwatx) [inline solutions/cses-1196.mdx:27]: usaco.guide coding conventions state that we need to use the following template:

```cpp
//BeginCodeSnip{C++ Short Template}
#include <bits/stdc++.h> // see /general/running-code-locally
using namespace std;

using ll = long long;

using vi = vector<int>;
#define pb push_back
#define all(x) begin(x), end(x)
#define sz(x) (int) (x).size()

using pi = pair<int,int>;
#define f first
#define s second
#define mp make_pair

void setIO(string name = "") {
	cin.tie(0)->sync_with_stdio(0); // see /general/fast-io
	if (sz(name)) {
		freopen((name + ".in").c_str(), "r", stdin); // see /general/input-output
		freopen((name + ".out").c_str(), "w", stdout);
	}
}
//EndCodeSnip

int main() {
  setIO();

}
```

@SansPapyrus683 do you want to detemplify this

**Review by nwatx — CHANGES_REQUESTED** [review]

File: solutions/cses-1196.mdx [hunk 7]
```diff
@@ -2,14 +2,94 @@
...
+		int u, v, w;
+		cin>>u>>v>>w;
+		u--, v--;
+		graph[u].push_back({v, w});
+	}
+	VVL dis = dijkstra(graph, 0, k);
+	for(auto x: dis[n-1]){
+			cout<<x<<" ";
+		}
+}
```
Comment (nwatx) [inline solutions/cses-1196.mdx:64]: just make this main

File: solutions/cses-1196.mdx [hunk 8]
```diff
@@ -2,14 +2,94 @@
...
+		
+		for(auto z: graph[node]){
+			if((int)dis[z.first].size() < k){
+				pq.insert({w + z.second, z.first});
+			}
+		}
+	}
+	return dis;
+}
+void solve(){
```
Comment (nwatx) [inline solutions/cses-1196.mdx:51]:
```suggestion
void solve() {
```

File: solutions/cses-1196.mdx [hunk 9]
```diff
@@ -2,14 +2,94 @@
...
+		u--, v--;
+		graph[u].push_back({v, w});
+	}
+	VVL dis = dijkstra(graph, 0, k);
+	for(auto x: dis[n-1]){
+			cout<<x<<" ";
+		}
+}
+int main()
+{
```
Comment (nwatx) [inline solutions/cses-1196.mdx:66]: bracing style not consistent

**Review by nwatx — CHANGES_REQUESTED** [review]

**Pull request closed without merging**